backend/docs/API.md

```diff
@@ -172,7 +172,9 @@ For `size` = 2
       "minprice": 30,
       "userid": 1,
       "categoryid": 2,
-      "timestamp": "2019-06-13T10:28:42.411752Z"
+      "timestamp": "2019-06-13T10:28:42.411752Z",
+      "telephone": "0000-0000",
```

Comment:
Could you please add the expiration time in these examples too? Thanks for the help!

Reply:
Did it in commit 7e6c693